_posts/2023-09-27-trino-gateway.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Trino Gateway has arrived"
+author: Manfred Moser, Martin Traverso
+image: /assets/images/logos/trino-gateway-small.png
+excerpt_separator: <!--more-->
+show_pagenav: false
+---
+
+You started with one Trino cluster, and your users like the power for SQL and
+[querying all sorts of data sources]({{site_url}}/ecosystem.html#data-sources).
+Then you needed to upgrade and got a cluster for testing going. That was a while
+ago, and now you run a separate cluster configured for ETL workloads with
+fault-tolerant execution, and some others with different configurations.
+
+With Trino Gateway we now have an answer to your users request to provide on URL
```

Comment:
I would also add a usecase of cluster availability during upgrades and switching to an upgrades cluster with ease.